src/transformers/models/electra/modeling_electra.py

```diff
@@ -1092,16 +1094,19 @@ def forward(
         Examples:
 
         ```python
-        >>> from transformers import ElectraTokenizer, ElectraForPreTraining
+        >>> from transformers import ElectraForPreTraining, ElectraTokenizerFast
         >>> import torch
 
-        >>> tokenizer = ElectraTokenizer.from_pretrained("google/electra-small-discriminator")
-        >>> model = ElectraForPreTraining.from_pretrained("google/electra-small-discriminator")
-
-        >>> input_ids = torch.tensor(tokenizer.encode("Hello, my dog is cute", add_special_tokens=True)).unsqueeze(
-        ...     0
-        >>> )  # Batch size 1
-        >>> logits = model(input_ids).logits
+        >>> discriminator = ElectraForPreTraining.from_pretrained("google/electra-small-discriminator")
+        >>> tokenizer = ElectraTokenizerFast.from_pretrained("google/electra-small-discriminator")
+        >>> sentence = "The quick brown fox jumps over the lazy dog"
+        >>> fake_sentence = "The quick brown fox fake over the lazy dog"
+        >>> fake_tokens = tokenizer.tokenize(fake_sentence)
+        >>> fake_inputs = tokenizer.encode(fake_sentence, return_tensors="pt")
+        >>> discriminator_outputs = discriminator(fake_inputs)
+        >>> predictions = torch.round((torch.sign(discriminator_outputs[0]) + 1) / 2)
+        >>> [print("%7s" % token, end="") for token in fake_tokens]
+        >>> [print("%7s" % int(prediction), end="") for prediction in predictions.squeeze().tolist()]
```

Comment:
Could we please add the expected output here as well for completeness/clarity?

Reply:
Sure